contracts/NodeDelegator.sol

```diff
@@ -207,11 +218,11 @@ contract NodeDelegator is INodeDelegator, LRTConfigRoleChecker, PausableUpgradea
     function getAssetBalance(address asset) public view override returns (uint256 ndcAssets, uint256 eigenAssets) {
         ndcAssets += IERC20(asset).balanceOf(address(this));
 
-        if (asset == WETH_TOKEN_ADDRESS) {
+        if (asset == WETH) {
             // Add any ETH in the NDC that was earned from execution rewards
             ndcAssets += address(this).balance;
 
-            eigenAssets = stakedButNotVerifiedEth;
+            eigenAssets += stakedButNotVerifiedEth;
```

Comment:
Is there a reason for this change to `+=`? Both the WETH path and the LST path only set `eigenAssets` once, but one now uses `+=` and the other `=`.